gradle.properties

```diff
@@ -6,8 +6,8 @@ POM_URL=https://github.com/airbnb/lottie-android
 POM_SCM_URL=https://github.com/airbnb/lottie-android
 POM_SCM_CONNECTION=scm:git@github.com:airbnb/lottie-android.git
 POM_SCM_DEV_CONNECTION=scm:git@github.com:airbnb/lottie-android.git
-POM_LICENSE_NAME=Apache License 2.0
-POM_LICENSE_URL=https://github.com/airbnb/lottie-android/blob/master/LICENSE
+POM_LICENSE_NAME=Apache-2.0
```

Comment:
I don't think this change is "more correct". Searching through GitHub, I see lots of projects that use `Apache license 2.0` and almost none that use `Apache-2.0`. `The Apache Software License, Version 2.0` actually seems to be the most common.

Reply:
Per https://maven.apache.org/pom.html#Licenses this change is bringing it in line with the new recommendations. Specifically,

> Using an [SPDX identifier](https://spdx.org/licenses/) as the license name is recommended.